NurVo_Frontend/src/pages/Home.tsx

```diff
@@ -49,60 +37,150 @@ const MenuTitle = ({ text, onPress }: { text: string, onPress: () => void }) =>
   );
 }
 
-function UserInfoHeader() {
+interface UserInfo {
+  id: string;
+  name: string;
+  nickname: string;
+  phone: string;
+  obj: string;
+  obj_date: string;
+}
+
+async function isFirstLogin(userId: string): Promise<boolean> {
+  try {
+    const value = await AsyncStorage.getItem(`firstLogin:${userId}`);
+    if (value === null) {
+      await AsyncStorage.setItem(`firstLogin:${userId}`, 'false');
+      return true;
+    }
+    return false;
+  } catch (e) {
+    console.log(e);
+    return false;
+  }
+}
+function UserInfoHeader({ numOfReview }: { numOfReview: number }) {
   const navigation = useNavigation();
+
+  const [userdata, setUserdata] = useState({});
+  const [progress, setProgress] = useState(0);
+  const [dueDate, setDueDate] = useState(0);
+  const [attendance, setAttendance] = useState<boolean[]>([]);
+
+  useFocusEffect(
+    useCallback(() => {
+      async function getUserData() {
+        try {
+          const user = await fetchMypage();
+          setUserdata(user);
+          setDueDate(getDueDate(user.obj_date));
+        } catch (error) {
+          console.error('Error fetching user info:', error);
+        }
+      }
+      getUserData();
+    }, [])
+  );
+
+  function formatDate(date: string) {
+    const dateObject = new Date(date);
+    const year = dateObject.getFullYear();
+    const month = (dateObject.getMonth() + 1).toString().padStart(2, "0");
+    const day = dateObject.getDate();
   
+    const formattedDate = `${year}.${month}.${day}`;
+    return formattedDate;
+  }
+
+  //최초 로그인 검사 -> 목표가 설정되어 있는지 확인
+  useEffect(() => {
+    async function checkFirstLogin() {
+      if((userdata.obj === null || userdata.obj === "") && (userdata.obj === null || userdata.obj === "")) {
```

Comment:
if((userdata.obj === null || userdata.obj === "") && (userdata.obj === null || userdata.obj === "")) {
```suggestion
      if((userdata.obj === null || userdata.obj === "") || (userdata.obj_date === null || userdata.obj_date === "")) {
```
obj 이 2개가 들어가 있어서요!! 혹시 모르니, obj_date 도 추가하면 좋을 거 같아요!